### ISSUE TITLE
Remount 

### ISSUE THREAD

**Issue by ishu0786** [markieeee/mu1ti_path#2]
Thanks for this got it working in first try. 

Btw have you got the remount code for this ios 11.3.x ?

**Comment by markie-dev** [comment]
nope, waiting for a developer to release it. Then I will probably implement it.

glad the jb worked :)
